src/components/StylesFromLiteralsService.js

```diff
@@ -0,0 +1,154 @@
+(function() {
+  goog.provide('ga_styles_from_literals_service');
+
+  var module = angular.module('ga_styles_from_literals_service', []);
+
+  module.provider('gaStylesFromLiterals', function() {
+
+    this.$get = function() {
+
+      function getOlStyleForPoint(options, shape) {
+        if (shape === 'circle') {
+          return new ol.style.Circle(options);
+        } else if (shape === 'icon') {
+          return new ol.style.Icon(options);
+        } else {
+          var shapes = {
+            square: {
```

Comment:
@procrastinatio support has been added for regular shapes:
https://github.com/geoadmin/mf-geoadmin3/commit/85f33fc99f000024bb6960452ed7f5b4b5ce6003